core/unit_test/cuda/TestCuda_InterOp_StreamsMultiGPU.cpp

```diff
@@ -72,7 +82,7 @@ TEST(cuda_multi_gpu, managed_views) {
   }
 }
 
-TEST(cuda_multi_gpu, unmanaged_views) {
+TEST_F(TEST_CATEGORY_FIXTURE(MultiGPU), unmanaged_views) {
```

Comment:
What does that expand to when you list the tests in the executable?

Reply:
```
$ ./Kokkos_CoreUnitTest_CudaInterOpStreamsMultiGPU --gtest_list_tests

cuda_MultiGPU.
  managed_views
  unmanaged_views
  scratch_space
  stream_sync_semantics_raw_cuda
```

Reply:
Then maybe `TEST_CATEGORY_FIXTURE(multi_gpu), ...)` would be more consistent

Reply:
Not blocking, just a suggestion

Reply:
I went this way since `TEST_CATEGORY_FIXTURE(MultiGPU)` is the name of a struct and I don't like the `_`, but I guess `TEST_CATEGORY_FIXTURE` is adding `cuda_` so I'm not actually avoiding that. And it match `TEST_F(TEST_CATEGORY_FIXTURE(GraphInterOp)`.

I don't mind reverting. I'll do that when I fix the build warning Daniel posted.